src/components/nav-user/nav-user.tsx

```diff
@@ -0,0 +1,107 @@
+import { Component, Element, h, Host, Prop, Event, EventEmitter } from "@stencil/core";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-nav-user",
+  styleUrl: "nav-user.scss",
+  shadow: {
+    delegatesFocus: true
+  }
+})
+export class CalciteNavUser {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteNavUserElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+  /** When true, the component is highlighted. */
+  @Prop({ reflect: true }) active: boolean;
+
+  /** Specifies the text to display, such as a username or full name. */
+  @Prop() fullName: string;
+
+  /** Describes the appearance of the avatar. If no label is provided, context will not be provided to assistive technologies. */
+  @Prop() label: string;
+
+  /** Specifies the unique id of the user. */
+  @Prop() userId: string;
+
+  /** Specifies the username of the user. */
+  @Prop() username: string;
```

Comment:
Sidebar: we should probably revisit this way in the future, but the casing of `username` and `fullName`/`full-name` does stand out when both are used. I get the reasoning, but something I noticed.

Reply:
Taken from Avatar so should address there as well if needed.